tests/conftest.py

```diff
@@ -91,7 +101,7 @@ def fin():
         # On teardown we wish to report on pods that were restarted by k8s during the test
         restarted_pods = pod.check_for_restarted_pods(testconfig['namespace'])
         if restarted_pods:
-            print('\n\nAttention!!! The following pods were restarted during test: {0}\n\n'.format(restarted_pods))
+            print('\n\nAttention!!! The following pods were restarted{1} test: {0}\n\n'.format(restarted_pods, S))
```

Comment:
I don't really get why you did that ?

Reply:
me neither, seems like I did it by mistake.
reverting

Reply:
fixed in 76c0de737d2899de92f55112ed664490f87cfba8